### PR TITLE
Refactor logging

### PR DESCRIPTION
**Pull request by cburmeister**
SSIA

### CODE REVIEW

**Comment by cburmeister** [comment]
An example from either log:

```
INFO:__init__.py:214 - Querying device status
DEBUG:connectionpool.py:387 - "GET /ssdp/device-desc.xml HTTP/1.1" 200 1089
DEBUG:socket_client.py:764 - Receiver:Updating status
DEBUG:socket_client.py:547 - Sending: Message urn:x-cast:com.google.cast.tp.connection from sender-0 to receiver-0: {'origin': {}, 'userAgent': 'PyChromecast', 'type': 'CONNECT', 'senderInfo': {'browserVersion': '44.0.2403.30', 'version': '15.605.1.3', 'connectionType': 1, 'platform': 4, 'sdkType': 2, 'systemVersion': 'Macintosh; Intel Mac OS X10_10_3'}}
DEBUG:socket_client.py:547 - Sending: Message urn:x-cast:com.google.cast.receiver from sender-0 to receiver-0: {'type': 'GET_STATUS', 'requestId': 1}
DEBUG:socket_client.py:243 - Connected!
DEBUG:socket_client.py:418 - Received: Message urn:x-cast:com.google.cast.receiver from receiver-0 to sender-0: {u'status': {u'volume': {u'muted': False, u'level': 1.0}, u'applications': [{u'displayName': u'Backdrop', u'statusText': u'', u'transportId': u'web-0', u'sessionId': u'8FA0CFAD-5074-4BB9-BE8F-EF945CC46336', u'namespaces': [{u'name': u'urn:x-cast:com.google.cast.sse'}], u'appId': u'E8C28D3C'}]}, u'type': u'RECEIVER_STATUS', u'requestId': 1}
DEBUG:socket_client.py:890 - Received status: CastStatus(is_active_input=False, is_stand_by=True, volume_level=1.0, volume_muted=False, app_id=u'E8C28D3C', display_name=u'Backdrop', namespaces=[u'urn:x-cast:com.google.cast.sse'], session_id=u'8FA0CFAD-5074-4BB9-BE8F-EF945CC46336', transport_id=u'web-0', status_text=u'')
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET / HTTP/1.1" 200 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/style.css HTTP/1.1" 304 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/tmp/fury.2014.1080p.mp4.jpg HTTP/1.1" 304 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/tmp/spectre.2015.1080p.mp4.jpg HTTP/1.1" 304 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/tmp/stop.making.sense.1984.1080p.mp4.jpg HTTP/1.1" 304 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/tmp/the.royal.tenenbaums.2001.1080p.mp4.jpg HTTP/1.1" 304 -
INFO:_internal.py:87 - 10.0.0.3 - - [21/Mar/2016 00:50:26] "GET /static/tmp/winter's.bone.2010.720p.mkv.jpg HTTP/1.1" 304 -
```

**Comment by dbolt** [comment]
If an error occurs we won't know when it happened? When I see something weird happen on my browser or phone I'd like to look at the servers logs to see what happened. But without a time-stamp I won't know if that just occurred or happened 4 weeks ago. And if the situation is hard to re-produce then I need to tail the logs while the event occurs to determine if that is the error message. Is that worth not having extra code? If the code is bothersome then move it to an external file to move it out of mind.

I'm applying things from other languages / stacks that may not apply here and looking for feedback in this regard.

**Comment by dbolt** [comment]
Just realized you have a formatter :) just add the time to it and I'm set.

File: media.py [hunk 1]
```diff
@@ -26,11 +26,10 @@ def __init__(self, filename):
         self.data = self.get_data()
         self.get_poster()
 
-
+    @property
```
Comment (dbolt) [inline media.py:29]: Oh that's sweet. Is this convention for getters? if I have an instance variable of "foo" is it convention to keep that variable private and then add?

``` python
class PropertyExample:

    def __init__(self):
        self._foo = 'foo bar string'

    @property
    def getFoo(self):
        return self._foo
```

Comment (cburmeister) [inline media.py:29]: The `@property` decorator is a shortcut to create read only properties

**Pull request merged**